src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -700,6 +709,85 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
 
             Console::success("Build id: $buildId created");
 
+            if ($resource->getCollection() === 'sites') {
+                try {
+                    $rule = Authorization::skip(fn () => $dbForPlatform->findOne('rules', [
+                        Query::equal("projectInternalId", [$project->getInternalId()]),
+                        Query::equal("resourceType", ["deployment"]),
+                        Query::equal("resourceInternalId", [$deployment->getInternalId()])
+                    ]));
+
+                    if($rule->isEmpty()) {
+                        throw new \Exception("Rule for build not found");
+                    }
+
+
+                    $client = new FetchClient();
+                    $client->addHeader('Authorization', 'Bearer ' . App::getEnv('_APP_OPENSSL_KEY_V1', ''));
```

Comment:
I would avoid using this key again, we over use it. I think we can do same as we do for the assistant container.

Reply:
Changed to use new env var, `_APP_BROWSER_SECRET`

Reply:
Removed secret all-together